lib/iex/lib/iex.ex

```diff
@@ -255,6 +255,28 @@ defmodule IEx do
   Number of expressions and their results to keep in the history.
   The value is an integer. When it is negative, the history is unlimited.
 
+  ## Documentation Helpers
+
+  Documentation Helpers allows iex to have a configurable set of modules
+  for looking up both Elixir and Erlang documentation. `:doc_helpers` is
+  a keyword list with these values.
+
+    * `:helpers` - A list of modules that implement the documentation
+                   functions. The default is `[IEx.H_Elixir, IEx.H_Erlang_Stub]`.
+                   Removing `IEx.H_Elixir` is not recommended.
+    * `:find`    - Can have two values `:first` or `:all` (:first is default)
+                   Determines whether all helpers are called or just
+                   the first helper that returns `:found`
```

Comment:
we are not using backtick in documentation around atoms; since it is understood that they are values.
so the line above can read
> Can have two values :first or :all (:first is default)

what I'm not sure, is that if we should use backticks as the head of the item, such as in `:helpers` or `:find`, maybe @josevalim has a say on this?